docker_templates/templates/docker_images_ros2/source/create_ros_image.Dockerfile.em

```diff
@@ -72,17 +72,17 @@ RUN pip3 install -U \
 @[  end if]@
 @[end if]@
 @
-@[if 'vcs' in locals()]@
-@[  if vcs]@
-
 @[if 'rosdep' in locals()]@
+
 # bootstrap rosdep
 @[  if 'rosdistro_index_url' in rosdep]@
 ENV ROSDISTRO_INDEX_URL @(rosdep['rosdistro_index_url'])
 @[  end if]@
 RUN rosdep init \
     && rosdep update
 @[end if]@
+@[if 'vcs' in locals()]@
+@[  if vcs]@
```

Comment:
arguably we could also move this conditional lower if we consider that the creation of the workspace directory should be done regardless.

This PR justs restore the scope to what it was prior to https://github.com/osrf/docker_templates/pull/44/files

Reply:
If that where the case, then perhaps a devel and source image for ros2 could share more of the same template?

Reply:
It could, I guess it depends what we would call a "devel" image ([discussed here](https://github.com/osrf/docker_templates/pull/49#issuecomment-474555091)).

Do you mean to always create a workspace (empty in the case of the devel image), and consumers will clone their own code in it ?

Reply:
> Do you mean to always create a workspace (empty in the case of the devel image), and consumers will clone their own code in it ?

I like that. https://github.com/osrf/docker_templates/pull/49#issuecomment-474635769

Reply:
Sounds good, I restricted the scope more in https://github.com/osrf/docker_templates/pull/51/commits/cea3e56af8bb556d3fe1a900fad88edf34ddd1b9. I'll make the other changes necessary for the `ros2:devel` image in a separate PR